scripts/install/install_macos.sh

```diff
@@ -272,6 +278,36 @@ check_prereqs()
   decrease_indent
 }
 
+# Test connection to BindPlane if it was specified
+connection_check()
+{
+  info "Help Me Obi-Wan Kenobi!"
+  if [ $check_bp_url == "true" ] ; then
+    if [ -n "$opamp_endpoint" ]; then
+      HTTP_ENDPOINT="$(printf "${opamp_endpoint}" | sed 's#ws#http#' | sed 's#/v1/opamp$##')"
```

Comment:
Shellcheck also hits this line with https://www.shellcheck.net/wiki/SC2059

Reply:
Good catch, and you're right it would nuke any of them. I'll fix that in my next revision.

Reply:
Well, I thought it would nuke them, but...
```
echo "wss://some-wss-url" | sed 's#ws#http#'                   
https://some-wss-url
```

Tested on both OSX and Linux.

That said, still putting in explicit protection.